internal/api/shared/schemas/schemas.go

```diff
@@ -115,32 +114,20 @@ func (s *sch) ValidatePanels(panels map[string]json.RawMessage) error {
 		value := s.context.CompileBytes(panelJSON)
 
 		// retrieve the corresponding panel schema
-		panelSchema, err := retrieveSchemaForKind(panelName, value, kindField, s.panels.schemas)
+		panelSchema, err := retrieveSchemaForKind(panelName, value, "spec.plugin.kind", s.panels.schemas)
```

Comment:
yeah it's much better to have the complete path as an hard coded string. It's definitely easier to read